### ISSUE TITLE
about generator loss graph

### ISSUE THREAD

**Issue by chenxianghu** [Justin-Tan/generative-compression#12]
i use the same code as you for training cityscapes dataset 60 epochs, the generator loss at last 23.163, but your model will decline to about 0.5
i use the defaut configuration in conf.py, only modify the num_epochs, i don't know why?

Epoch 59 | Generator Loss: **23.163** | Discriminator Loss: 0.410 | Rate: 0 examples/s (99773.67 s)

**Comment by Justin-Tan** [comment]
High generator loss is not necessarily a bad thing because there are a lot of additional terms in the generator loss function I have added since the initial commit. Is the generated image quality poor?

**Comment by chenxianghu** [comment]
@Justin-Tan  The generated images quality almost the same as yours on cityscapes dataset, so i think i don't need to care the Generator Loss according to your reply. Thank you very much!
